src/main/java/org/apache/bcel/classfile/LineNumber.java

```diff
@@ -96,15 +96,15 @@ public void dump( final DataOutputStream file ) throws IOException {
      * @return Corresponding source line
      */
     public int getLineNumber() {
-        return 0xffff & lineNumber;
+        return lineNumber & 0xffff;
```

Comment:
Is this a style change? If not a comment is needed.

Reply:
style change